### PR TITLE
[fix/edit] Add namespace field in getstarted.md

### PR DESCRIPTION
**Pull request by rahulchheda**
Signed-off-by: Rahul M Chheda <rahul.chheda@mayadata.io>

### CODE REVIEW

**Review by ajeshbaby — DISMISSED** [review]
LGTM

File: docs/getstarted.md [hunk 1]
```diff
@@ -102,6 +102,7 @@ apiVersion: v1
 kind: ServiceAccount
```
Comment (chandankumar4) [inline docs/getstarted.md:102]: Can you update permission from the respective experiment, i.e: `container-kill`, and the name of service account is also mismatch

Comment (ksatchit) [reply docs/getstarted.md:102]: lets also ensure the permissions are same

Comment (rahulchheda) [reply docs/getstarted.md:102]: Done. @ksatchit @chandankumar4

**Review by ksatchit — APPROVED** [review]

**Review by chandankumar4 — APPROVED** [review]

**Pull request merged**